package.json

```diff
@@ -7,30 +7,31 @@
     "build": "webpack --config webpack.prod.js",
     "preinstall": "bin/preinstall.js"
   },
-  "dependencies": {},
-  "devDependencies": {
+  "dependencies": {
     "bourbon": "~4.3",
     "bourbon-neat": "~1.9",
-    "copy-webpack-plugin": "^5.1.1",
-    "css-loader": "^3.1.0",
-    "file-loader": "^3.0.1",
     "font-awesome": "^4.7.0",
     "html5shiv": "^3.7.3",
-    "imports-loader": "^0.8.0",
     "jquery": "^3.4.1",
     "lato-font": "^3.0.0",
+    "roboto-fontface": "^0.10.0",
+    "wyrm": "^1.0.9"
```

Comment:
Technically, all of these are dev dependencies as they are only required to build the static assets. I don't think we're directly depending on any of these currently, are we?

Reply:
Technically the theme.js requires jquary however reading npm docs I think this pr is still correct usage. Splitting them still shows what affects built assets an what does not.

https://docs.npmjs.com/specifying-dependencies-and-devdependencies-in-a-package-json-file specifies dev requirements as dependencs needed for local development and testing.

If we update fontawesome, it is a required change to the product, updating webpack does not affect the the end product.

I dont think npm is super strict in this regard but I prefer this organization between what is needed to make the theme and what is needed to actually build the theme.